### PR TITLE
Hamburger Menu

### PR DESCRIPTION
**Pull request by kennethlien**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

Customer phone number screen now has this funny three stripe thing on the top left, opens up a weird tab thing on the left side that happens to have two buttons that go to the same link and maybe even a logout button.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Online sources

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review

Most changes are in AppNavigator.js. New component is DrawerButton, which is only used in CustomerLookupScreen.

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots

Simulator Screen Shot - iPad (7th generation) - 2020-03-22 at 21 47 05

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @tommypoa @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: navigation/AppNavigator.js [hunk 1]
```diff
@@ -1,28 +1,124 @@
...
+            height: 114,
+            display: 'flex',
+            flexDirection: 'row',
+            alignItems: 'flex-end',
+            padding: 16
+          }}>
+          {console.log(this.state.clerkName)}
+          <Title style={{ color: 'white' }}>{this.state.clerkName}</Title>
+        </View>
+        <TouchableOpacity style={{ padding: 16 }} onPress={() => Linking.openURL('tiny.cc/SubmitFeedbackClerk')}>
```
Comment (wangannie) [inline navigation/AppNavigator.js]: Needs http://! This throws an error

File: components/DrawerButton.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <ButtonContainer style={{ ...style }} onPress={() => navigation.toggleDrawer()}>
+      <FontAwesome5 name="bars" size={24} color={light ? Colors.lightest : Colors.activeText} />
+    </ButtonContainer>
+  );
+}
+
+DrawerButton.propTypes = {
+  navigation: PropTypes.object.isRequired,
+  light: PropTypes.bool.isRequired,
```
Comment (tommypoa) [inline components/DrawerButton.js]: Seems like you followed the prop rules from `BackButton.js` but thie light prop is not necessary here since the hamburger stripes will always be dark!
